string pick_room
string ob_name
string drop_room
